source/includes/_10-one-time-sync-company-custom-check.html.md.erb

```diff
@@ -0,0 +1,547 @@
+
+#### Check type
+
+The `check_type` field must be set to `"COMPANY_CUSTOM"`.
+
+#### Check template
+
+The `check_template` fields for this check are:
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Always Present?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>type</code></td>
+      <td>The value <code>"ONE_TIME_SYNCHRONOUS"</code></td>
+      <td>Yes</td>
+      <td>
+        Which check template to use for this check. For one-time synchronous
+        checks, you must specify <code>"ONE_TIME_SYNCHRONOUS"</code>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>timeout</code></td>
+      <td>number</td>
+      <td>No</td>
+      <td>
+        The number of seconds PassFort should wait without a response before
+        considering identity checks run through this integration to have
+        timed out. Must be an integer. If not specified or set to
+        <code>null</code>, a timeout of 60 seconds will be used.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<%= partial("includes/partials/authenticated.md.erb", :locals => {
+  :role => :server,
+}) %>
+
+## Run One-Time Sync Company Custom Check
+
+This endpoint exposes your data provider's check results to PassFort.
+
+### HTTP Request
+
+`POST https://my-integration.example.com/checks`
+
+>A JSON payload following this structure will be sent to the endpoint:
+
+```json
+<%= pretty_json("requests/run_check_sync_company_custom", "json") %>
+```
+
+The payload of the request can contain the following fields:
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Always present?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>id</code></td>
+      <td>UUID</td>
+      <td>Yes</td>
+      <td>
+        Every check instruction will contain a unique ID which can be used to
+        track and identify individual requests without needing to use other
+        information in the request.
+      </td>
+    </tr>
+    <tr>
+      <td><code>demo_result</code></td>
+      <td>
+        One of the <a href="#demo-result-types">specified demo result types</a>
+      </td>
+      <td>No</td>
+      <td>
+        If this field is present, the check must be considered a demo check
+        and you should respond with appropriate demo data. See the
+        <a href="#demo-checks">Demo Checks section</a> for more information.
+      </td>
+    </tr>
+    <tr>
+      <td><code>commercial_relationship</code></td>
+      <td><code>"DIRECT"</code></td>
+      <td>Yes</td>
+      <td>
+        Specified whether the check is for a customer with a direct
+        relationship with your integration's data provider, or if they are
+        using the check through PassFort's pay-as-you-go reselling scheme. 
+        Reselling is not available for Company Custom Check
+      </td>
+    </tr>
+    <tr>
+      <td><code>check_input</code></td>
+      <td>A CompanyData object</td>
```

Comment:
I think we should format "CompanyData" as code since it's referring to a specific object